INTERNALS.md

```diff
@@ -211,3 +201,8 @@ All timestamps are represented as integers, milliseconds since the Unix epoch.
 
   Job identifiers are formatted as batch UIDs.
 
+`counters` (hash)
+
+  Telemetry counters.
```

Comment:
Yep, that's it. To be clear: gauges = instant value of something; counters = accumulators (value only ever goes up, or gets reset)